prog/dftb+/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ if(WITH_DFTD3)
 endif()
 
 if(WITH_TRANSPORT)
-  target_link_libraries(dftbplus PUBLIC negf sparskit mudpack)
```

Comment:
Is this on top of the changes in #463 ?

Reply:
This is the usual problems I have with submodules... I was working on another branch, then I checked out the libnegf head which requires this change and kept it when creating this new branch. If we do not really like it here I can checkout the old libnegf revision. I was wandering if I can squash some commit in the github remote to cleanup history.

Reply:
If its merged after #463 there shouldn't be a problem.